test/e2e/storage/utils/create.go

```diff
@@ -141,14 +141,7 @@ func PatchItems(f *framework.Framework, items ...interface{}) error {
 // - only the latest stable API version for each item is supported
 func CreateItems(f *framework.Framework, items ...interface{}) (func(), error) {
 	var destructors []func() error
-	var cleanupHandle framework.CleanupActionHandle
```

Comment:
The description of `CreateItems` must be adapted. It currently still says:
```
// Cleaning up after a test can be triggered in two ways:
// - the test invokes the returned cleanup function,
//   usually in an AfterEach
// - the test suite terminates, potentially after
//   skipping the test's AfterEach (https://github.com/onsi/ginkgo/issues/222)
```

The new behavior is that there is only one way of deleting the created objects.

Reply:
it can still get called from `AfterSuite` hooks via  - https://github.com/kubernetes/kubernetes/pull/90773/files#diff-4d73861d7cac72f86d4c7bccdc4e594fR226 though.